onchain/Dockerfile

```
@@ -48,7 +48,7 @@ RUN yarn build
 RUN npm install -g .
 
 # this stage is runtime image for rollups (hardhat)
-FROM base as hardhat
+FROM node:16.17.0-bullseye-slim as hardhat
```

Could we define an `ARG` to hold the node version?

```suggestion
FROM node:${NODEJS_VERSION}-bullseye-slim as hardhat
```